apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -115,6 +115,13 @@
 			</exclusions>
 		</dependency>
 
+		<dependency>
```

Comment:
Please note that BFD already had this dependency: it was getting it transitively from `jessentials`. Now that that's gone, we're just including it directly. No SIA should be needed.